### ISSUE TITLE
command variables

### ISSUE THREAD

**Issue by jmcbee** [doowb/watch-cli#2]
How do I detect in my command which file has changed?

**Comment by doowb** [comment]
watch-cli doesn't pass anything to the command. If you have any ideas on how that would work, I'd be interested in seeing them.

**Comment by jmcbee** [comment]
maybe setting an environment variable as the command gets called.

**Comment by thomasfl** [comment]
My pull request solves this problem by making the environment variables FILENAME, EVENT and  ABSOLUTE_FILENAME available in the command:

Linux/macOS:

```
$ watch -p '**/*.js' -c 'jshint $FILENAME'
```

In Windows:

```
> watch -p "**/*.js" -c "jshint %FILENAME%"
```

**Comment by doowb** [comment]
Closing since this was released in `0.2.3`. Thanks @thomasfl for the PR: #8

**Comment by thomasfl** [comment]
You’re welcome Brian. :-)
